src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -495,7 +495,7 @@ public virtual bool RunInMemory
         public bool CoraxIncludeSpatialDistance { get; set; }
         
         [Description("The maximum amount of memory that Corax can use for a memoization clause during query processing")]
-        [DefaultValue(128)]
+        [DefaultValue(512)]
```

Comment:
Is it intentional change?

Reply:
Shouldn't this value be different in the case of 32 bits?

Reply:
Yes, it is. We need to have a high amount until we fix https://issues.hibernatingrhinos.com/issue/RavenDB-21245

This fixes: https://issues.hibernatingrhinos.com/issue/RavenDB-21220